examples/customresources/models/mybase/mybase.go

```diff
@@ -130,6 +130,10 @@ func (base *MyBase) IsMoving(ctx context.Context) (bool, error) {
 	return false, nil
 }
 
+func (base *MyBase) Properties(ctx context.Context, extra map[string]interface{}) (map[base.Feature]float64, error) {
+	return nil, errUnimplemented
```

Comment:
very minor, not a blocker: i think we should have a dummy implementation (or some comments) here. this example will be used by people to implement new drivers and a nudge in the direction of implementing this method would be good imo. granted, we don't this on other methods but i feel those are more intuitive than this.

Reply:
let me know if you like what I added